res/database/Default/database_scripts.xml

```diff
@@ -1975,6 +1976,419 @@ Ekscytująca rozrywka i łamiące mózg pytania quizowe.</pl>
+					</description>
+				</scripttemplate>
+				<scripttemplate index="24" product="7" licence_type="2" guid="17d3ebbc-75bd-43a2-af80-c08a20422d3c">
+					<title>
+						<de>Warum leuchten Glühwürmchen?</de>
+						<en>Why do fireflies glow?</en>
+						<pl>Dlaczego świetliki świecą?</pl>
+					</title>
+					<description>
+						<de>${.self:"role":1:"firstname"} staunt: Glühwürmchen zaubern mit Chemie! Sie mischen Sauerstoff mit speziellen Stoffen und zünden ein kleines Licht an - für die große Liebe oder zur Warnung!</de>
+						<en>${.self:"role":1:"firstname"} is amazed: Fireflies use chemistry! They mix oxygen with special chemicals to light up - for true love or as a warning signal!</en>
+						<pl>${.self:"role":1:"firstname"} się zachwyca: Świetliki używają chemii! Mieszają tlen ze specjalnymi substancjami i zapalają światełko - dla miłości lub jako ostrzeżenie!</pl>
+					</description>
+				</scripttemplate>
+				<scripttemplate index="25" product="7" licence_type="2" guid="097216ef-c207-47c6-b84d-7d7b34032713">
+					<title>
+						<de>Warum schnarchen Menschen?</de>
+						<en>Why do people snore?</en>
+						<pl>Dlaczego ludzie chrapią?</pl>
+					</title>
+					<description>
+						<de>${.self:"role":1:"firstname"} hört's genau: Beim Schlafen entspannen sich Muskeln. Die Luft pfeift durch die lockere Kehle und rüttelt alles durch. Ein schnarchendes „Konzert“ beginnt!</de>
+						<en>${.self:"role":1:"firstname"} listens closely: Sleeping muscles relax. Air whistles through the floppy throat and shakes things up. A snoring “concert” begins!</en>
+						<pl>${.self:"role":1:"firstname"} nasłuchuje: Podczas snu mięśnie się rozluźniają. Powietrze świszczy przez luźne gardło i wszystko wprawia w wibracje. Zaczyna się „koncert” chrapania!</pl>
+					</description>
+				</scripttemplate>
+				<scripttemplate index="26" product="7" licence_type="2" guid="bb4ef536-3d03-4ff2-ab22-0946c70b259d">
+					<title>
+						<de>Warum knackt es in den Fingern?</de>
+						<en>Why do fingers crack?</en>
+						<pl>Dlaczego palce strzelają?</pl>
+					</title>
+					<description>
+						<de>${.self:"role":1:"firstname"} macht, dass es knackt: Beim Ziehen der Finger entsteht ein Gasbläschen im Gelenk, das platzt. Ein kleines „Plopp“ - aber keine Sorge, alles bleibt heil!</de>
+						<en>${.self:"role":1:"firstname"} makes it pop: Pulling fingers forms a gas bubble in the joint, which bursts. A little "pop" - but don't worry, it's all fine!</en>
+						<pl>${.self:"role":1:"firstname"} strzela palcami: Przy rozciąganiu palców powstaje bąbelek gazu, który pęka. Małe „plop” - ale spokojnie, nic się nie psuje!</pl>
+					</description>
+				</scripttemplate>
+				<scripttemplate index="27" product="7" licence_type="2" guid="e9e67487-33f7-4ef9-ae5a-ec49e6e1966f"> <!-- 'dupe' but other explanation -->
+					<title>
+						<de>Warum fliegen Flugzeuge?</de>
```

Comment:
yeah but different description. Noticed that too and commented. All in all repetitions cannot be ruled out for the more dynamic templates either. So leaving as is.

Reply:
The issue is, that a "random selection" can randomly (heh...) select both "variants" for one series. Read: you might end up with a series of 5 episodes in which 2 are "duplicates" (variants).

For now there are only two options to avoid things:
- possible now: create 2 scrips with each of them only containing 1 variant (but of course this reduces "variety" if you also split up the ones only existing in one variation)
- requiring efforts: flags/markers telling that an episode cannot "co-exist" with others. You would eg. mark the variants as to exclude each other. This can be useful if you eg want to offer multiple "variants" of a series pilot.

Ok there is another: removing the duplicates (and saving them for later)